SnapkitPractice/SnapkitPractice/ViewController.swift

```diff
@@ -0,0 +1,141 @@
+//
+//  ViewController.swift
+//  SnapkitPractice
+//
+//  Created by 이중엽 on 2023/03/06.
+//
+
+import UIKit
+import SnapKit
+
+enum FirstName: Int {
+    case kim = 0
```

Comment:
이미 알고있을 수도 있지만 작성해둡니다ㅎㅎ
rawValue를 Int 타입으로 지정해줄 때, 첫번째 case를 0 부터 시작할 거라면 대입해주지 않아도 됩니다!
```suggestion
    case kim
```

Reply:
수정 완료했습니다!